tests/integration/test_backward_compatibility/test_functions.py

```diff
@@ -13,7 +13,11 @@
 backward = cluster.add_instance(
     "backward",
     image="clickhouse/clickhouse-server",
-    tag="22.9",
+    # Note that a bug changed the string representation of several aggregations in 22.9 and 22.10 and some minor
+    # releases of 22.8, 22.7 and 22.3
+    # See https://github.com/ClickHouse/ClickHouse/issues/42916
+    # Affected at least: singleValueOrNull, last_value, min, max, any, anyLast, anyHeavy, first_value, argMin, argMax
+    tag="22.6",
```

Comment:
I wish I choose 22.6 (release without backported bug) here initially and then the problem will be discovered earlier...

Reply:
Welp. I hope we'll increase test coverage in the future to catch these problems, but at least now we have a start.

Reply:
I have one idea in mind: run multiple servers at once with difference version (patched and previous release), same as backward compatibility check but will use different servers at once, will take a look once will finish with other stuff.